cmd/lncli/commands.go

```diff
@@ -1986,20 +1986,20 @@ var updateChannelPolicyCommand = cli.Command{
 				"with a granularity of 0.000001 (millionths). Can not " +
 				"be set at the same time as fee_rate_ppm.",
 		},
-		cli.Uint64Flag{
+		cli.StringFlag{
 			Name: "fee_rate_ppm",
 			Usage: "the fee rate ppm (parts per million) that " +
 				"will be charged proportionally based on the value of each " +
 				"forwarded HTLC, the lowest possible rate is 0 " +
 				"with a granularity of 0.000001 (millionths). Can not " +
 				"be set at the same time as fee_rate.",
 		},
-		cli.Int64Flag{
+		cli.StringFlag{
 			Name: "time_lock_delta",
 			Usage: "the CLTV delta that will be applied to all " +
 				"forwarded HTLCs",
```

Comment:
I think an easy quick solution is to add a `Value` field here, sth like `Value: 4`?